### ISSUE TITLE
Behavior change for guessed UTC in 0.5.15

### ISSUE THREAD

**Issue by craigforster** [moment/moment-timezone#598]
The latest release has changed the behavior when UTC is configured on the server.  Previously, UTC was reported as `Etc/UTC` and now it's reported as `Etc/Universal`.  This breaks downstream dependencies that rely on on detecting `Etc/UTC` (for better or worse), for example https://github.com/gautamsi/ews-javascript-api.

Reproduction script:

guess.js:
```
const moment = require('moment-timezone');
console.log(moment.tz.guess());
```

With 0.5.14:

```
$ TZ=Etc/UTC node guess.js
Etc/UTC
```

With 0.5.15:

```
$ TZ=Etc/UTC node guess.js
Etc/Universal
```

This looks to be due to the change in the data files, the 2017c version contains `"Etc/UTC|UTC|0|0|",` whereas the 2018d version contains `"Etc/Universal|UTC|0|0|",`.

**Comment by ellenaua** [comment]
@craigforster Thanks! I Just released 0.5.16 with a fix.

**Comment by craigforster** [comment]
Thanks for the quick fix!
